src/LanguageServer/Impl/LanguageServer.Configuration.cs

```diff
@@ -73,23 +73,23 @@ public sealed partial class LanguageServer {
         private void HandleDiagnosticsChanges(JToken pythonSection, LanguageServerSettings settings) {
             var analysis = pythonSection["analysis"];
 
-            settings.diagnosticPublishDelay = GetSetting(analysis, "diagnosticPublishDelay", 1000);
+            settings.diagnosticPublishDelay = GetSetting(analysis, "diagnosticPublishDelay", defaultValue: 1000);
             var ds = _services.GetService<IDiagnosticsService>();
             ds.PublishingDelay = settings.diagnosticPublishDelay;
 
             ds.DiagnosticsSeverityMap = new DiagnosticsSeverityMap(
-                GetSetting(analysis, "errors", Array.Empty<string>()),
-                GetSetting(analysis, "warnings", Array.Empty<string>()),
-                GetSetting(analysis, "information", Array.Empty<string>()),
-                GetSetting(analysis, "disabled", Array.Empty<string>()));
+                GetSetting(analysis, "errors", defaultValue: Array.Empty<string>()),
```

Comment:
Not entirely convinced in need to specify defaults explicitly, makes it longer. There are VS extensions that display then if needed.

Reply:
it makes reviewing code easier though :) especially when constant is used. but not a strong argument. but if team prefer no named argument, I can go and delete those.